Source/NSubstitute.Acceptance.Specs/MultipleThreads.cs

```diff
@@ -112,6 +112,35 @@ public void Create_Delegate_Substitute_From_Many_Threads()
             Task.AwaitAll(tasks);
         }
 
+        [Test]
```

Comment:
@zvirja I think it's enough to have only multi-threaded test in our case. We have pletty of tests for `Returns(...)`